pwndbg/gdblib/symbol.py

```diff
@@ -211,7 +211,7 @@ def address(symbol: str) -> int:
         # If we try to look up a TLS variable when there is no TLS, this
         # exception occurs. Ideally we should come up with a way to check for
         # this case before calling `gdb.lookup_symbol`
-        skipped_exceptions.append("Cannot find thread-local variables")
+        skipped_exceptions.append("Cannot find thread-local")
```

Comment:
@gsingh93, will it have any problem if we check `"Cannot find thread-local"` instead of `"Cannot find thread-local variables"`?

Reply:
I guess it should work fine, merging for now, @gsingh93 let us know if u see any issues with that